.github/workflows/e2e.yaml

```diff
@@ -93,35 +93,12 @@ jobs:
         # output_dir
         mkdir -p _artifacts
         # create cluster
-        cat <<EOF | /usr/local/bin/kind create cluster \
+        sed -i  's/IP_FAMILY/'${IP_FAMILY}'/g' examples/kind/kind.config
```

Comment:
nit: consider using shell variables directly in the config, and the `envsubst` utility to fill them.